ui/admin/app/templates/scopes/scope/workers/worker/tags.hbs

```diff
@@ -61,11 +62,78 @@
                 <Hds::Badge @text={{B.data.type}} />
               {{/if}}
             </B.Td>
+            <B.Td>
+              {{#if (eq B.data.type this.tagTypeApi)}}
+                <Hds::Dropdown as |dd|>
+                  <dd.ToggleIcon
+                    @icon='more-horizontal'
+                    @text='Overflow Options'
```

Comment:
This should be a translated string

Reply:
This one actually isn't seen by the user and is only there because it is required. Here is [Hds documentation](https://helios.hashicorp.design/components/dropdown?tab=code#toggleicon-as-overflow-menu) on its use. Happy to create a translation for this but decided not to since it isn't seen.

Reply:
It says it's still used for aria-label which means screen reader users will still be using it no?

Reply:
I would vote, unless is an internal or machine attribute (such as `data-test-*`), everything else that we send for rendering I will suggest being under intl.

Reply:
Translation added.